### DISCUSSION TITLE
Add release to list

### DISCUSSION THREAD

**Discussion by captain-cronch** [joalla/discogs_client#132] [Ideas] [answered]
I see that it’s possible to add a release to user collection folders, but is it possible to add releases to user lists?

I currently have a script that checks an input song/artist against my collection and prints releases that contain that song by the artist, but if my collection doesn’t contain any matching releases, it would be nice to have potential releases that match be put into a list for me to check later.

Forgive me if this is currently possible, nothing I found in the documentation made it seem as if you could add to a list, only query one.

**Comment by JOJ0** [comment, marked as answer]
https://www.discogs.com/developers/#page:user-lists

is what Discogs API provides. They dont seem to have a create-api :-( Please doublecheck! I might not have searched well enough. 

Thanks for reaching out and sorry we cant help much :-/

**Reply by JOJ0** [reply]
This is what you've used for fetching user lists, right? https://python3-discogs-client.readthedocs.io/en/latest/discogs_client.html#discogs_client.models.List
